cdci_data_analysis/analysis/parameters.py

```diff
@@ -398,10 +416,23 @@ def check_type(par_type, allowed, name):
     @staticmethod
     def check_value(val, units=None, name=None, par_format=None):
         pass
-
+    
+    @staticmethod
+    def check_bounds(val, min_value, max_value, name):
+        raise NotImplementedError(f"Parameter {name} doesn't support min_value/max_value check")
+        
     def reprJSONifiable(self):
         # produces json-serialisable list
         reprjson = [dict(name=self.name, units=self.units, value=self.value)]
+        restrictions = {}
```

Comment:
note that `check_value` is also a restriction. It can even be used by the plugin developer to implement this functionality. Could you please check what `check_value` does in plugins, and what should be the remaining function of `check_value`?

Reply:
I don't see any uses of `check_value` in the plugins. 
In the derived Parameter classes, it's generally used to check for the proper python data type of the value.

Reply:
To be clear, there are no direct uses of it in plugins. 
There is one indirect use https://github.com/oda-hub/dispatcher-plugin-integral/blob/0f8d9754aa7c3da5a6aba14d8fb4c8acab9f60f7/cdci_osa_plugin/osa_isgri.py#L55
https://github.com/oda-hub/dispatcher-plugin-integral/blob/0f8d9754aa7c3da5a6aba14d8fb4c8acab9f60f7/cdci_osa_plugin/osa_isgri.py#L55
to check for the bounds.

So I propose to remove check_value from the specific parameters' signature. There are allowed_values and min/max for the most useful checks. If a plugin developer really needs some complex check, it could be accomplished with inheritance. 

Anyway, it's unclear how to report complex checks in parameter metadata.

Reply:
In principle, it makes sense. But to be safe, I'd make a deprecation warning when attempting to use it, instead of removing.

Reply:
Resolved by https://github.com/oda-hub/dispatcher-app/pull/493/commits/82f3cb5ed5e1ad34886970330efd84fb3080011f